### ISSUE TITLE
Hard disk requirement for uyuni server

### ISSUE THREAD

**Issue by mbussolotto** [uyuni-project/uyuni-docs#2446]
Documentation says:
https://github.com/uyuni-project/uyuni-docs/blob/16a26820b15677f9643c77e519e69a06e45bc3f8/modules/installation-and-upgrade/pages/uyuni-install-requirements.adoc?plain=1#L43-L44

but 
https://github.com/uyuni-project/uyuni/blob/4517565ca83375660a2d326fd73fb0cdb23634b6/susemanager/yast/susemanager_reqs.rb#L115

## Documentation Update Required

Misaligment betwen Documentation and product warning.

## Subject Matter Expert (SME)

@mbussolotto

**Comment by stdevel** [comment]
IMHO the documentation should mention that additional ~50 GB are realistic when using Package Hub - most users mirror this repository.

**Comment by keichwa** [comment]
> IMHO the documentation should mention that additional ~50 GB are realistic when using Package Hub - most users mirror this repository.

Makes sense.  I'll add such a note.

**Comment by keichwa** [comment]
>     * `/var/spacewalk` should be at least 100 GB (the check is hardcoded)
> 
>     * SUSE product and package hub requires 50GB
> 
>     * 360 GB per Red Hat product

Thanks I updated the PR.  Please review if time permits!

**Comment by stdevel** [comment]
> > ```
> > * SUSE product and package hub requires 50GB
> > ```
> 

Thanks for your efforts! To me, this sounds a bit like both SUSE products *and* Package Hub requires in summary 50 GB. Maybe we can rephrase this to become more clear? Maybe `SUSE products and Package Hub both requires 50 GB respectively`? Just an idea.

**Comment by keichwa** [comment]
For 5.0 and containerized uyuni, we simplified it:

```
|
| [path]``/var/lib/containers/storage/volumes``
| Minimum 150 GB (depending on the number of products)
...
```
With the explanation:
> As long as external storage is mounted to [path]``/var/lib/containers/storage/volumes``, {productname} does not need or use storage on the [literal]``/`` partition, but leaves that to the management of the container host itself.

If needed, we can improve it in the future.
